From 076fbe061333bdfecbd5765c782c477233e38e2f Mon Sep 17 00:00:00 2001
From: John Morris <john@zultron.com>
Date: Tue, 5 Jan 2021 23:30:10 -0600
Subject: [PATCH] resolv.conf.5: tfix

Change `RES_USE_EDNSO` to `RES_USE_EDNS0`, defined in
`resolv.h`.  (This is written correctly in `man3/resolver.3` in this
same repo.)  Helps with grepping and internet searches!

Signed-off-by: John Morris <john@zultron.com>
Signed-off-by: Michael Kerrisk <mtk.manpages@gmail.com>
---
 man5/resolv.conf.5 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man5/resolv.conf.5 b/man5/resolv.conf.5
index c1b3d017a..8ed958932 100644
--- a/man5/resolv.conf.5
+++ b/man5/resolv.conf.5
@@ -264,7 +264,7 @@ these options were removed in glibc 2.25.
 .TP
 .BR edns0 " (since glibc 2.6)"
 Sets
-.BR RES_USE_EDNSO
+.BR RES_USE_EDNS0
 in
 .IR _res.options .
 This enables support for the DNS extensions described in RFC\ 2671.
-- 
2.37.3

